### PR TITLE
docs: update roadmap

### PR DESCRIPTION
**Pull request by pi0**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/content/5.community/4.roadmap.md [hunk 1]
```diff
@@ -54,6 +52,6 @@ Module         | Status              | Nuxt Support | Repository | Description
 ---------------|---------------------|--------------|------------|-------------------
 Content 1.x    | Maintenance         | 2.x          | [nuxt/content](https://github.com/nuxt/content/tree/v1) | Maintenance only
 Content 2.x    | Active              | 3.x          | [nuxt/content](https://github.com/nuxt/content) | Released
-Auth           | WIP                 | 3.x          | [nuxt/auth](https://github.com/nuxt/auth) | Private Beta
-Image          | Active              | 2.x          | [nuxt/image](https://github.com/nuxt/image) | Nuxt 3 support planned
-Telemetry      | Active              | 2.x          | [nuxt/telemetry](https://github.com/nuxt/telemetry/) | Nuxt 3 support planned
+Auth           | WIP                 | 3.x          | [nuxt/auth](https://github.com/nuxt/auth) | Nuxt 3 support is planned to work after session support
+Image          | Active              | 2.x          | [nuxt/image](https://github.com/nuxt/image) | Nuxt 3 support and is work in progress: [nuxt/image#500](nuxt/image/issues/500)
```
Comment (danielroe) [inline docs/content/5.community/4.roadmap.md:56]:
```suggestion
Image          | Active              | 2.x          | [nuxt/image](https://github.com/nuxt/image) | Nuxt 3 support is in progress: [nuxt/image#500](nuxt/image/issues/500)
```

File: docs/content/5.community/4.roadmap.md [hunk 2]
```diff
@@ -54,6 +52,6 @@ Module         | Status              | Nuxt Support | Repository | Description
 ---------------|---------------------|--------------|------------|-------------------
 Content 1.x    | Maintenance         | 2.x          | [nuxt/content](https://github.com/nuxt/content/tree/v1) | Maintenance only
 Content 2.x    | Active              | 3.x          | [nuxt/content](https://github.com/nuxt/content) | Released
-Auth           | WIP                 | 3.x          | [nuxt/auth](https://github.com/nuxt/auth) | Private Beta
-Image          | Active              | 2.x          | [nuxt/image](https://github.com/nuxt/image) | Nuxt 3 support planned
-Telemetry      | Active              | 2.x          | [nuxt/telemetry](https://github.com/nuxt/telemetry/) | Nuxt 3 support planned
+Auth           | WIP                 | 3.x          | [nuxt/auth](https://github.com/nuxt/auth) | Nuxt 3 support is planned to work after session support
```
Comment (danielroe) [inline docs/content/5.community/4.roadmap.md:55]:
```suggestion
Auth           | WIP                 | 3.x          | [nuxt/auth](https://github.com/nuxt/auth) | Nuxt 3 support is planned after session support
```

**Comment by pi0** [comment]
Thanks for fixes @danielroe <3

File: docs/content/5.community/4.roadmap.md [hunk 3]
```diff
@@ -12,8 +12,8 @@ Nuxt is constantly evolving, with new features and modules being added all the t
 
 Release             | Expected date   | Description
 --------------------|-----------------|---------------------------------------------------
-`nuxt@2.16`         | April,   2022   | Nuxt v2 cumulative updates for future compatibility
-`nuxt@3.0.0`        | June,    2022    | Nuxt v3 stable release
+`nuxt@2.16`         | Midsummer ,  2022   | Nuxt v2 cumulative updates for future compatibility with Bridge
```
Comment (A5rocks) [inline docs/content/5.community/4.roadmap.md:15]: There's an extra space here, by the way. (It should be "Midsummer," not "Midsummer ,")

**Comment by ibrahimBeladi** [comment]
I think the use of mid-summer or any seasonal keywords is misleading to half the people due to having 2 hemispheres.
I would suggest using Q1/Q2/Q3/Q4 or ranges instead, rather than hemispheres-specific timing.

**Pull request merged**